Pomodoro.Tests/DataAccessTests/CompletedRepositoryTests.cs

```diff
@@ -0,0 +1,241 @@
+﻿// <copyright file="CompletedRepositoryTests.cs" company="PomodoroGroup_GL_BaseCamp">
+// Copyright (c) PomodoroGroup_GL_BaseCamp. All rights reserved.
+// </copyright>
+
+using Pomodoro.DataAccess.EF;
+using Pomodoro.DataAccess.Entities;
+using Pomodoro.DataAccess.Repositories.Realizations;
+using Pomodoro.Tests.EqualityComparers;
+
+namespace Pomodoro.Tests.DataAccessTests
+{
+    /// <summary>
+    /// Compled repository test class.
+    /// </summary>
+    public class CompletedRepositoryTests
+    {
+        /// <summary>
+        /// Adds completed to database.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> object that represents an asynchronous operation.</returns>
+        [Fact]
+        public async Task AddAsync_AddsCompletedToDatabase()
+        {
+            // arrange
+            using var context = new AppDbContext(UnitTestHelper.GetUnitTestDbOptions());
```

Comment:
better to use Moq(it's a library), and you can write smth like context = new Mock(new DbContext), this allows you to verify that some methods were called.